run.py

```diff
@@ -15,20 +15,59 @@
 """
```

Comment:
You should probably update the comments here to give the user a brief idea of what kinds of commands they can run. I believe you have some outdated text here.